### PR TITLE
feat: 'MergeRequestOpened' and 'MergeRequestMerged'

### LINKED ISSUE #152: Add GitLab support for MR merged events
docs: https://docs.gitlab.com/ee/user/project/integrations/webhooks.html

# Implementation plan
- [x] new `/gitlab` endpoint #153 
- [x] new `MergeRequestMerged` reaction for GitLab #156 
- [ ] add `MergeRequestMerged` reaction to `/gitlab` endpoint #160
~new "`Ping`" reaction for GitLab~

### LINKED ISSUE #155: Add GitLab support for MR opened events
docs: https://docs.gitlab.com/ee/user/project/integrations/webhooks.html

Implementation plan
- [x] new `MergeRequestOpened` reaction for GitLab #156 
- [ ] add `MergeRequestOpened` reaction to `/gitlab` endpoint #160

### PR DESCRIPTION
**Pull request by SantiMA10**
Resolves #152 
Resolves #155 

🙏🏻

### CODE REVIEW

File: test/routes/gitlab/merge-request-hook.spec.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+				.getInstance();
+
+			const { result } = await subject.inject({
+				method: 'POST',
+				url: '/gitlab',
+				payload,
+				headers: { 'x-gitlab-event': 'Merge Request Hook' },
+			});
+
+			expect(result).toEqual(expect.objectContaining({ messages: expect.anything() }));
```
Comment (orestes) [inline test/routes/gitlab/merge-request-hook.spec.ts:33]: This test is not making any meaningful checks.
Should we at least check the message content?

This test would pass even if we were ignoring the event.

File: test/routes/gitlab/merge-request-hook.spec.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
```
Comment (orestes) [inline test/routes/gitlab/merge-request-hook.spec.ts:49]: Same as before. This test is not making any meaningful checks.
Should we at least check the message content?

This test would pass even if we were ignoring the event.

File: test/routes/gitlab/merge-request-hook.spec.ts (hunk 1 above)
Comment (SantiMA10) [reply test/routes/gitlab/merge-request-hook.spec.ts:33]: No, when we ignore the event we send a `message`, that's why I'm only checking that contains a `messages` key

Comment (SantiMA10) [reply test/routes/gitlab/merge-request-hook.spec.ts:33]:
```plain
● POST /gitlab › Merge Request Hook › handles merge request merged event

    expect(received).toEqual(expected) // deep equality

    Expected: ObjectContaining {"messages": Anything}
    Received: {"message": "Ignoring event: 'Merge Request Hook'"}
```

File: test/routes/gitlab/merge-request-hook.spec.ts (hunk 2 above)
Comment (SantiMA10) [reply test/routes/gitlab/merge-request-hook.spec.ts:49]: Same comment here

**Pull request merged**